src/tools/splash2txt/CMakeLists.txt

```diff
@@ -160,6 +165,8 @@ add_executable(splash2txt
      ${SRCFILES}
      )
 
+target_link_libraries(splash2txt Splash::Splash)
```

Comment:
does `Splash::Splash` not add `zlib`?

Reply:
No it is not adding zlib. We do it in PIConGPU also separate.

Reply:
ok, then we should report this as an issue in libSplash

Reply:
I don't even know why we depend on zlib in Splash, we don't have it in our interfaces... Probably it's a missing dependency in HDF5 already that we overlink for sanity.

Reply:
Just did testing on hypnos: we don't need to link zlib here.

Reply:
I tested it on hemera and the zlib is required. Keep in mind that maybe hypnos modules are older and maybe compiled against the system zlib.

Reply:
this is the error message without zlib:

```
CMake Error at CMakeLists.txt:164 (add_executable):
  Target "splash2txt" links to target "ZLIB::ZLIB" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
````

Reply:
Jep, this is already overlinked by libSplash: https://github.com/ComputationalRadiationPhysics/libSplash/issues/283